docs/UserGuide.md

```diff
@@ -75,26 +75,46 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a volunteer: `add` [COMING SOON]
 
-Adds a person to the address book.
+Adds a volunteer to an event.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add vn/NAME hp/PHONE_NUMBER e/EMAIL /to en/EVENT`
 
-<box type="tip" seamless>
-
-**Tip:** A person can have any number of tags (including 0)
-</box>
+Parameters:
+* vn/ - Volunteer name
+* hp/ - Volunteer phone number
+* e/ - Volunteer email
+* en/ - Event to be added in
 
```

Comment:
Perhaps the restrictions can be inserted here to align with the current AB3 format?